src/bare_item.rs

```diff
@@ -0,0 +1,209 @@
+use crate::{Num, Parser};
+use std::{convert::TryFrom, fmt, ops::Deref};
+
+// TODO: Necessary? rust_decimal seems to validate already
+// pub struct Decimal(rust_decimal::Decimal);
```

Comment:
I'm a bit torn apart here between "unnecessary to add, it's the same thing" and "keep it consistent". Any preference?